.github/scripts/notify_discord.py

```
@@ -16,31 +26,63 @@
-        ["gh", "api", f"repos/{repo}/commits/{sha}/pulls", "--jq", ".[0] // empty"],
-        capture_output=True, text=True, check=True,
+def send(webhook_url, content, mention_role_id=""):
+    """Send a message to a Discord webhook."""
+    req = urllib.request.Request(
+        webhook_url,
+        data=json.dumps({
+            "content": content,
+            "allowed_mentions": {"roles": [mention_role_id]} if mention_role_id else {},
+        }).encode(),
+        headers={"Content-Type": "application/json", "User-Agent": "SneezyMUD-CI"},
     )
```

Ruff (0.15.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)